### PR TITLE
Release/4.1.0 beta.3

### PR DESCRIPTION
**Pull request by ElektrojungeAtWork**
@ranterle @lumaxis Can you guys please do a quick sanity check of the changes to the readme and changelog? =)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -453,6 +453,40 @@ metricsManager.trackEventWithName(eventName)
...
+                                 @"Measurement 2" : @2.34,
+                                 @"Measurement 3" : @2000000};
+
+[metricsManager trackEventWithName:eventName properties:myProperties measurements:myMeasurements]
+```
+
+**Swift**
+
+```swift
+
```
Comment (lumaxis) [inline README.md]: Empty row here is probably unnecessary?

File: HockeySDK-Source.podspec [hunk 2]
```diff
@@ -25,7 +25,7 @@ Pod::Spec.new do |s|
   s.frameworks              = 'AssetsLibrary', 'CoreGraphics', 'CoreTelephony', 'CoreText', 'MobileCoreServices', 'Photos', 'QuartzCore', 'QuickLook', 'Security', 'SystemConfiguration', 'UIKit'
   s.libraries = 'c++', 'z'
   s.vendored_frameworks = 'Vendor/CrashReporter.framework'
-  s.pod_target_xcconfig     = {'GCC_PREPROCESSOR_DEFINITIONS' => %{$(inherited) BITHOCKEY_VERSION="@\\"#{s.version}\\"" BITHOCKEY_C_VERSION="\\"#{s.version}\\"" BITHOCKEY_BUILD="@\\"60\\"" BITHOCKEY_C_BUILD="\\"60\\""} }
+  s.pod_target_xcconfig     = {'GCC_PREPROCESSOR_DEFINITIONS' => %{$(inherited) BITHOCKEY_VERSION="@\\"#{s.version}\\"" BITHOCKEY_C_VERSION="\\"#{s.version}\\"" BITHOCKEY_BUILD="@\\"85\\"" BITHOCKEY_C_BUILD="\\"85\\""} }
```
Comment (lumaxis) [inline HockeySDK-Source.podspec]: Should be `95`

**Comment by lumaxis** [comment]
@TroubleMakerBen A couple of minor comments, looks good otherwise.

**Comment by TroubleMakerBen** [comment]
Thx @lumaxis =) committing the changes

**Pull request closed without merging**